desktop/package.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const os = require('os')
+const webpack = require('webpack')
+const cfg = require('./webpack.config.production.js')
+const packager = require('electron-packager')
+const del = require('del')
+const exec = require('child_process').exec
+const argv = require('minimist')(process.argv.slice(2))
+const devDeps = Object.keys(require('./package.json').devDependencies)
+const fs = require('fs-extra')
+
+const appName = 'Keybase'
+const shouldUseAsar = argv.asar || argv.a || false
+const shouldBuildAll = argv.all || false
+const appVersion = argv.appVersion || '0.0.0'
+const comment = argv.comment || ''
+
+del.sync('build')
+
+fs.copySync('./Icon.png', 'build/desktop/Icon.png')
```

Comment:
For now we explicitly copy assets over, in the future we'll likely want to consolidate where all this stuff goes (like /assets or something)